fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/Main.java

```diff
@@ -638,7 +639,7 @@ protected void createFhirSchemas() {
      * into the FHIR resource tables
      */
     protected void updateProcedures() {
-        if (!MULTITENANT_FEATURE_ENABLED.contains(dbType)) {
+        if (!MULTITENANT_FEATURE_ENABLED.contains(dbType) || DbType.POSTGRESQL == dbType) {
```

Comment:
Add a different set and use the contains. 
Call it STORED_PROCEDURE_ENABLED and add to the set. 
This is very hard to figure out if we constantly add.

Reply:
good idea!